src/command/render/latexmk/texlive.ts

```diff
@@ -285,14 +286,33 @@ function tlmgrCommand(
   args: string[],
   _quiet?: boolean,
 ) {
+  let cmdExec: string[];
+  let tempFile: string | undefined;
+  if (Deno.build.os === "windows") {
+    // writing to a file to exectute to avoid quoting issue with Deno
+    // https://github.com/quarto-dev/quarto-cli/issues/336
+    const lines = ["tlmgr", cmd, ...args];
+    tempFile = Deno.makeTempFileSync(
+      { prefix: "tlmgr-cmd", suffix: ".bat" },
+    );
+    Deno.writeTextFileSync(tempFile, lines.join(" ") + "\n");
+    cmdExec = ["cmd", "/c", tempFile];
+  } else {
+    cmdExec = ["tlmgr", cmd, ...args];
+  }
   try {
     const result = execProcess(
       {
-        cmd: [...tlmgr, cmd, ...args],
+        cmd: cmdExec,
         stdout: "piped",
         stderr: "piped",
       },
     );
+    result.finally(() => {
+      if (Deno.build.os === "windows" && tempFile) {
+        removeIfExists(tempFile);
+      }
+    });
```

Comment:
At first I tried adding a `finally` in the `tryCatch` but I think that is the right way to go as `execProcess()` is a promise and we want to remove the file if success or if it fails.